util/ipresolver/__init__.py

```diff
@@ -19,11 +19,15 @@
 from util.abchelpers import nooper
 
 ResolvedLocation = namedtuple(
-    "ResolvedLocation", ["provider", "service", "sync_token", "country_iso_code", "aws_region"]
+    "ResolvedLocation",
+    ["provider", "service", "sync_token", "country_iso_code", "aws_region", "continent"],
```

Comment:
"region" might be more accurate than "continent".

Reply:
I chose `continent` because that is the most consistent across the board. `region` is different for different people. For aws regions are like `us-east-1` and for RedHat regions are like APAC, NASA etc. Continent codes are pretty much standard.